src/cores/ads/AdsList.jsx

```diff
@@ -17,15 +20,22 @@ const AdsList = () => {
   const { isVisible, toggleModal, getId } = useModal();
 
   useEffect(() => {
-    (async () => { try {
-      const data = await inventoryServices.fetchAds(token, adsState);
-      dispatch(storeAllAds(data));
-    } catch (e) {
-      alert(e);
-    }
-  })()
-  }, [adsState.amount, adsState.page]);
-
+    (async () => {
+      try {
+        dispatch(setLoading(true));
+        const data = await inventoryServices.fetchAds(token, adsState);
+        dispatch(storeAllAds(data));
+      } catch (error) {
+        if (error.response) {
+          throw new Error(
+            error?.response?.data?.message || "Error obteniendo anuncios"
+          );
+        } toast.error(error.message);
```

Comment:
El:

```
if (error.response) {
          throw new Error(
            error?.response?.data?.message || "Error obteniendo anuncios"
          );
        }
```

Le corresponde al inventoryServices.fetchAds no al componente, el componente solo deberia mostrar el Toast, de hecho... ¿para que throweamos aqui si no se va a poder mostrar el error?

```suggestion
        toast.error(error.message);
```